### DISCUSSION TITLE
BTS Heavy Doors

### DISCUSSION THREAD

**Discussion by ImagineUsingHammer** [BEEmod/BEE2-items#4222] [Feature Suggestions]
I 100% remember they were in BEEmod, but they are just... _gone._ Is there any chance you can bring them back? They were really cool.

**Comment by FrozenDragon0** [comment]
This is something that's more reserved for a UCP, and I'm pretty sure there's a UCP that has them. There'd be no point putting these in the main mod as they are functionally the same as a normal test chamber door.
